src/screens/UserPortal/Organizations/Organizations.tsx

```
@@ -465,15 +467,22 @@
+                            data-country-code={organization.address.countryCode}
                           >
+                            {/* Directly insert the country code as text */}
+                            <span style={{ display: 'none' }}>
+                              {organization.address.countryCode}
+                            </span>
+
                             <div
                               data-testid={`membership-status-${organization.name}`}
                               data-status={organization.membershipRequestStatus}
                               className="visually-hidden"
                             ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)